specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -545,35 +545,784 @@ paths:
+        '409':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/409"
+        '410':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/410"
+        '415':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/415"
+        '429':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/429"
+        '500':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/500"
+        '503':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/503"
+      tags:
+      - Hypotheken
+  /hypotheken:
+    get:
+      operationId: GetHypotheken
+      description: |
+          Het zoeken van hypotheken van een specifieke hypotheekhouder die als geldverstrekker een recht van hypotheek vestigt als zekerheid voor de lening. Geef één van de volgende combinaties van parameters op :
+          1.  Hypotheekhouder is een Persoon die bekend is in de BRP als ingeschreven persoon
+              -  ingeschrevenpersoon__burgerservicenummer (verplicht)
+              -  persoon__typepersoon (verplicht)
+          2.  Hypotheekhouder is niet bekend als ingeschreven persoon in de BRP of de hypotheekhouder is een rechtspersoon
+              -  persoon__identificatie (verplicht)
+              -  persoon__typepersoon (verplicht)
+      parameters:
+        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/parameters/fields"
+        - in: query
+          name: ingeschrevenpersoon__burgerservicenummer
+          description: "Het burgerservicenummer van de ingeschreven persoon."
+          required: false
+          schema:
+            type: string
+            maxLength: 9
+        - in: query
+          name: persoon__identificatie
+          description: "De identificatie van een kadaster persoon of een rechtspersoon die bij het kadaster is geregistreerd."
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: persoon__typepersoon
+          description: "Met dit type wordt aangegeven of het een Persoon uit de Basisregistratie Personen (ingeschrevenpersoon) betreft danwel een persoon die door het kadaster wordt geregistreerd (kadasterpersoon). Een kadasterpersoon kan een persoon zijn die niet in de BRP bekend is of een Rechtspersoon."
+          required: true
+          schema:
+            $ref: "#/components/schemas/PersoonType_enum"
```

Comment:
Welke waarden kan deze parameter aannemen? Als ik het goed begrijp toch alleen maar 'ingeschreven_natuurlijk_persoon' en 'kadaster_natuurlijk_persoon'. Deze zijn toch 1 op 1 gerelateerd met de parameters 'ingeschrevenpersoon__burgerservicenummer' en 'persoon__identificatie'. In feite maak je daar dus al een keuze voor type persoon. Is de parameter 'persoon__typepersoon' dan nog wel noodzakelijk?

Reply:
ik denk dat bij gebruik van het burgerservicenummer het al duidelijk is welk type het is.

Alleen bij gebruik van de persoon__identificatie (kadasterpersoonidentificatie) kan het onduidelijk zijn of het een natuurlijk persoon of een niet-natuurlijk persoon is. Heeft de provider dit nodig om hiermee te zoeken?

Bij het zoeken van een onroerende zaak op persoon identificatie is persoon type immers ook niet nodig.

Hier is er ook de mogelijkheid als type ingeschreven_niet_natuurlijk_persoon te kiezen, wat suggereert dat je kan zoeken op rsin. Dat wordt voorlopig niet ondersteund.